src/components/ChatApp.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Header from './Header';
+import ChatMessages from './ChatMessages';
+import ChatInput from './ChatInput';
+import chatData from '../assets/data/ChatData.json'
+import userData from '../assets/data/UserData.json'; 
+
+const ChatContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 800px;
+  max-width: 400px;
+  margin: 0 auto;
+  border: 1px solid #ddd;
+  border-radius: 10px;
+  overflow: hidden;
+`;
+
+const ChatApp = () => {
+  const [messages, setMessages] = useState([]);
+  const [users, setUsers] = useState({});
+
+  useEffect(() => {
+    // JSON 파일로부터 데이터를 가져와 상태 설정
+    setMessages(chatData);
+    
+    // 사용자 데이터를 객체 형태로 변환하여 상태에 설정
+    const usersMap = {};
+    userData.forEach(user => {
+      usersMap[user.id] = user;
+    });
+    setUsers(usersMap);
+  }, []);
+
+  // 새로운 메시지를 추가하는 함수
+  const handleSendMessage = (newMessageText) => {
+    const newMessage = {
+      id: Date.now(), 
+      userId: 0,
+      time: new Date().toLocaleTimeString(), 
+      text: newMessageText 
+    };
```

Comment:
```suggestion
   const MY_ID = 0;


    const newMessage = {
      id: Date.now(), 
      userId: MY_ID,
      time: new Date().toLocaleTimeString(), 
      text: newMessageText 
    };
```
이런 경우 확장성을 고려해서 상수로 분리해서 사용하면 좋을 것 같아요!